tools/Azure.Mcp.Tools.Foundry/src/Options/FoundryOptionDefinitions.cs

```diff
@@ -49,7 +49,7 @@ public static class FoundryOptionDefinitions
         $"--{Endpoint}"
     )
     {
-        Description = "The endpoint URL for the Azure AI service.",
+        Description = "The endpoint URL for the Azure AI Foundry project/service.",
```

Comment:
Just confirming - was it intentional that we use service here instead of resource? I was comparing this with the terminology used in e2eTestPrompts.md.